x-pack/plugins/spaces/public/share_saved_objects_to_space/components/context_wrapper.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useState, useEffect, PropsWithChildren } from 'react';
+import { StartServicesAccessor, CoreStart } from 'src/core/public';
+import { createKibanaReactContext } from '../../../../../../src/plugins/kibana_react/public';
+import { PluginsStart } from '../../plugin';
+
+interface Props {
+  getStartServices: StartServicesAccessor<PluginsStart>;
+}
+
+export const ContextWrapper = (props: PropsWithChildren<Props>) => {
+  const { getStartServices, children } = props;
+
+  const [coreStart, setCoreStart] = useState<CoreStart>();
+
+  useEffect(() => {
+    Promise.all([getStartServices()]).then(([startServices]) => {
```

Comment:
Is there a reason to use `Promise.all` here, when we'll only ever have a single promise to resolve?

Reply:
Forgot to remove this while refactoring, thanks for pointing it out!